src/scripting/game_lua_kernel.cpp

```diff
@@ -1188,6 +1188,61 @@ namespace {
 	}
 }
 
+static int impl_end_level_data_get(lua_State* L)
+{
+	const end_level_data& data = *static_cast<end_level_data*>(lua_touserdata(L, 1));
+	const char* m = luaL_checkstring(L, 2);
```

Comment:
What is `m` used for?

Reply:
That's just the key being looked up. Every getter function calls it m… I think it's because the macros expect that?

I didn't write this code for the record, it was just copy-pasted from further down the file.